faq/dedibox-in-scaleway-console.mdx

```diff
@@ -95,17 +95,17 @@ In the future, you may only need to use your Scaleway login information to conne
 
 ## What impact will linking accounts have on my billing?
 
-You will continue to receive an invoice for each type of product (one for Dedibox and one for Scaleway Elements products).
-However, the default payment method defined in the Elements console will allow you to pay all your Scaleway invoices (Dedibox and Elements). It is important that you verify the default payment method defined in the Scaleway console after the linking process, to ensure the proper payment method will be used for your future invoices.<br />
+You will continue to receive an invoice for each type of product (one for Scaleway Dedibox and one for Scaleway Cloud products).
+However, the default payment method defined in the Scaleway console will allow you to pay all your Scaleway invoices (Scaleway Dedibox and Scaleway Cloud). It is important that you verify the default payment method defined in the Scaleway console after the linking process, to ensure the proper payment method will be used for your future invoices.<br />
```

Comment:
However, it is the payment method defined in your Scaleway console that will allow you to pay all your Scaleway invoices (Scaleway Dedibox and Scaleway Cloud). It is thus important that following the linking process, you verify whether the default payment method defined in the Scaleway console is your preferred payment method for future invoices.

Reply:
```suggestion
However, it is the payment method defined in your Scaleway console that will allow you to pay all your Scaleway invoices (Scaleway Dedibox and Scaleway Cloud). It is thus important that following the linking process, you verify whether the default payment method defined in the Scaleway console is your preferred payment method for future invoices.<br />
```